api/src/main/java/org/apache/iceberg/AppendFiles.java

```diff
@@ -42,16 +42,17 @@ public interface AppendFiles extends SnapshotUpdate<AppendFiles> {
    * <p>The manifest must contain only appended files. All files in the manifest will be appended to
    * the table in the snapshot created by this update.
    *
-   * <p>By default, the manifest will be rewritten to assign all entries this update's snapshot ID.
-   * In that case, it is always the responsibility of the caller to manage the lifecycle of the
-   * original manifest.
+   * <p>The manifest will be used directly if snapshot ID inheritance is enabled (all tables with
```

Comment:
Simply clarifies the notion of snapshot ID inheritance and that it is always on in V2 tables.